crates/lakekeeper/src/service/authz/implementations/allow_all.rs

```diff
@@ -240,4 +241,45 @@ impl Authorizer for AllowAllAuthorizer {
     async fn delete_view(&self, _view_id: ViewId) -> Result<()> {
         Ok(())
     }
+
+    async fn require_namespace_action(
```

Comment:
We should not implement these two methods (require_namespace_action & require_table_action) here - it only leeds to errors.
These methods are already implemented for `AllowAllAuthorizer` via the implementations in the `Authorizer` trait. The behaviour implemented here has no effect for the `OpenFGAAuthorizer` - only for `AllowAllAuthorizer`. Diverting behaviour is an anti-pattern and should only be applied if we have very good reasons for it - which I believe we don't.